src/main/java/com/vmtapp/enterprise/dao/TicketDAOStub.java

```diff
@@ -11,6 +11,12 @@
 
     List<Ticket> allTickets = new ArrayList<Ticket>();
 
+
+    /**
+     * Takes a ticket as a parameter to be saved
+     * @param ticket a ticket object to be saved
+     * @return return the saved ticket
+     */
```

Comment:
JavaDoc is a good idea.  I'd take it one step further, as most of what is here can be read from the method signature itself.  Only add JavaDoc if it is providing value over and above what can be read from the method.